api/app/Rules/PoolClosed.php

```diff
@@ -30,7 +30,7 @@ public function passes($attribute, $value)
     {
         $pool = Pool::find($value);
 
-        return $pool->closing_date->isFuture();
+        return is_null($pool->closing_date) || $pool->closing_date->isFuture();
```

Comment:
This file is named backwards, right?  It's a rule that the pool is NOT closed?

Reply:
Oops! 😅 Yeah, that was my mistake. You are correct, `PoolNotClosed` is what it should be.

Reply:
Opened #6770